nixos/roles/mailserver.nix

```diff
@@ -99,9 +101,41 @@ in
           List of virtual domains that this mail server serves. The first value
           is the canonical domain used to construct internal addresses in
           various places.
+
+          Finegrained control of settings is possible by setting
+          <literal>flyingcircus.roles.mailserver.domain."domain.tld".autoconfig = false;</literal>
         '';
       };
 
+      # this allows finegrained control over each domain
+      # for example domain."test.fcio.net".autoconfig = false;
+      domain = mkOption {
```

Comment:
can this be set in JSON config?

Reply:
```
{"domain":{"domain.tld":{"autoconfig":false}}}
```

Does it automatically take all options from JSON or anything specific necesarry?

Reply:
Tested, seems so it does it automagically